src/Common/Services/Mail/MailConfiguration.cs

```diff
@@ -1,17 +1,9 @@
+using Passwordless.Common.Services.Mail.Strategies;
+
 namespace Passwordless.Common.Services.Mail;
 
 public class MailConfiguration
 {
-    /// <summary>
-    /// The default email address to use as the sender.
-    /// </summary>
-    public string? From { get; set; }
-
-    /// <summary>
-    /// The default name to use as the sender.
-    /// </summary>
-    public string? FromName { get; set; }
```

Comment:
breaking-change: This would be a breaking change. Perhaps we can keep this and simply not require the "default channel" in the provider config?